package.json

```diff
@@ -19,31 +19,35 @@
     "Efim Bogdanovsky (https://github.com/lilsweetcaligula)"
   ],
   "scripts": {
-    "build": "tsc -d",
+    "build": "docker-compose build",
+    "start": "docker-compose up",
+    "stop": "docker-compose kill",
     "watch": "tsc -w -d",
     "test": "lab -v -P test -L -t 90 -r console -o stdout -r html -o test/coverage.html -r lcov -o test/lcov.info --sourcemaps --transform node_modules/lab-transform-typescript",
     "test-some": "lab -v -P test --sourcemaps --transform node_modules/lab-transform-typescript -g",
     "prettier": "prettier --write --no-semi --single-quote src/**/*.ts test/*.js",
-    "reset": "npm run clean && npm i && npm run build && npm test",
+    "reset": "npm run clean && npm i && npm test",
     "clean": "rm -rf node_modules dist package-lock.json yarn.lock",
     "repo-tag": "REPO_VERSION=`node -e \"console.log(require('./package').version)\"` && echo TAG: v$REPO_VERSION && git commit -a -m v$REPO_VERSION && git push && git tag v$REPO_VERSION && git push --tags;",
     "repo-publish": "npm run clean && npm i --registry=http://registry.npmjs.org && npm run repo-publish-quick",
     "repo-publish-quick": "npm run prettier && npm run build && npm test && npm run repo-tag && npm publish --access public --registry=https://registry.npmjs.org"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/senecajs/seneca-mem-store"
+    "url": "https://github.com/senecajs/seneca-knex-store"
   },
   "keywords": [
     "seneca",
     "plugin",
     "store",
-    "mem",
-    "memory"
+    "knex",
+    "knexjs"
   ],
   "peerDependencies": {
     "seneca": ">=3",
-    "seneca-entity": ">=18"
+    "seneca-entity": ">=18",
+    "knex": ">=2.4.2",
+    "pg": ">=8.8.0"
```

Comment:
pg should be a devDep - knex works with other dbs too

Reply:
True, in the future we can use knex-store to adapt to other DBs?